Marlin/language_cz.h

```diff
@@ -226,6 +226,7 @@
 #define MSG_CONTROL_RETRACT_RECOVER         _UxGT("UnRet mm")
 #define MSG_CONTROL_RETRACT_RECOVER_SWAP    _UxGT("S UnRet mm")
 #define MSG_CONTROL_RETRACT_RECOVERF        _UxGT("UnRet  V")
+#define MSG_CONTROL_RETRACT_SWAPF           _UxGT("Swap V") // need translation
```

Comment:
Remove these lines from all language files. We have other ways to point out what needs translation.